process_manager/templates/process_manager/partials/message_items.html

```diff
@@ -0,0 +1 @@
+{% for message in messages %}<li class="list-group-item">{{ message }}</li>{% endfor %}
```

Comment:
Maybe it's worth substituting a repeat delay `hx-get` into the bootstrap instant load `hx-get` in index, as you've done for the table?